src/probnum/quad/solvers/belief_updates/_belief_update.py

```diff
@@ -74,19 +128,28 @@ def __call__(
             kernel_means = np.concatenate(
                 (
                     bq_state.kernel_means,
-                    bq_state.kernel_embedding.kernel_mean(new_nodes),
+                    updated_kernel_embedding.kernel_mean(new_nodes),
                 )
             )
 
-        initial_integral_variance = bq_state.kernel_embedding.kernel_variance()
+        chol_gram = self._gram_cholesky_decomposition(gram)
+
+        # Update scaling parameter
+        updated_scale_sq = self._update_scale(fun_evals, chol_gram)
+
+        initial_integral_variance = updated_kernel_embedding.kernel_variance()
         weights = self._solve_gram(gram, kernel_means)
```

Comment:
This method was deleted. Does the code run? See also comment about `cho_factor` and `cho_solve`

Reply:
It does not run.